errors/error_handling.js

```diff
@@ -0,0 +1,19 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else next(err);
+};
+
+// handle specific psql errors
+exports.handlePsqlErrors = (err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "Invalid input" });
+  } else next(err);
+};
+
+// if the error hasn't been identified,
+// respond with an internal server error
+exports.handleServerErrors = (err, req, res, next) => {
+  console.log(err);
+  res.status(500).send({ msg: "Internal Server Error" });
```

Comment:
Aside from the 500 error, we don't need the 400 and 404 error handlers just yet. We only want to implement these on an endpoint that needs them and after testing said endpoint :)